navbar.php

```diff
@@ -19,12 +35,12 @@
         <li><a href="./gallery.php">GALLERY</a></li>
         <li><a href="./lastMeet.php">LAST MEET</a></li>
         <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Sponsors <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li><a href="./sponsors2012.php">2012</a></li>
-            <li><a href="./sponsors2013.php">2013</a></li>
-            <li><a href="./sponsors2014.php">2014</a></li>
-            <li><a href="./sponsors2015.php">2015</a></li>
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">SPONSORS <span class="caret"></span></a>
+          <ul class="dropdown-menu" style="background-color:#222 ; ">
+            <li><a href="./sponsors2012.php" class="drop-li" >2012</a></li>
```

Comment:
@uagarwal744 

Please remove the space between the closing `double quote` and the closing angular bracket.